src/components/Modal/IdentityConnectModal/IdentityConnectModal.tsx

```diff
@@ -38,7 +39,10 @@ function IdentityConnectModal() {
   }
 
   const checkProfile = useMemo(() => !timestamp || new Date() > new Date(timestamp), [timestamp])
-  const { isProfileValidated, validationChecked } = useIsProfileValidated(checkProfile ? user : null)
+  const { isProfileValidated, validationChecked } = useIsProfileValidated(
+    checkProfile ? user : null,
+    USER_FULLY_VALIDATED
```

Comment:
This variable `export const USER_FULLY_VALIDATED = [AccountType.Forum, AccountType.Discord]` to me is a list of all the "external" accounts a user can link to their address. 

IMHO we should call it smth like `ACCOUNT_TYPES`, or `LINKABLE_ACCOUNT_TYPES`.

I like the "fully validated" idea tho, we could use it for the result of the hook when called with all account types for example. (Could be `isUserFullyValidated`)